bynder_sdk/client/asset_bank_client.py

```diff
@@ -103,14 +103,15 @@ def delete_usage(self, integration_id, asset_id, query: dict = None):
             params=query
         )
 
-    def upload_file(self, file_path, brand_id, media_id: str = '', query: dict = None):
-        """ Upload file.
-        """
+    def upload_file(self, file_path: str, brand_id: str,
+                    media_id: str = '', query: dict = None):
+        """ Upload file."""
         if query is None:
             query = {}
         query['brandId'] = brand_id
-        self.upload_client.upload(
+        response = self.upload_client.upload(
             file_path=file_path,
             media_id=media_id,
             upload_data=query
         )
+        return response['mediaid']
```

Comment:
Im not sure if we want to return just the mediaid of the created/updated media or all the information.

Reply:
We should return everything, so this should be `return response` or just:
```
     return self.upload_client.upload(
        file_path=file_path,
        media_id=media_id,
        upload_data=query
    )
```